srcpkgs/spectacle/template

```diff
@@ -1,19 +1,20 @@
 # Template file for 'spectacle'
 pkgname=spectacle
-version=23.08.5
+version=24.02.0
 revision=1
 build_style=cmake
-configure_args="-DBUILD_TESTING=OFF"
-hostmakedepends="extra-cmake-modules kdoctools python3 qt5-host-tools qt5-qmake
- qt5-wayland kconfig gettext kcoreaddons wayland-devel"
-makedepends="kColorPicker-devel kImageAnnotator-devel kdeclarative-devel
- knewstuff-devel kwayland-devel libkipi5-devel purpose-devel
- xcb-util-cursor-devel xcb-util-image-devel kpipewire-devel
- pipewire-devel wayland-devel"
-depends="qt5-tools"
+configure_args="-DKF6_HOST_TOOLING=/usr/lib/cmake"
+hostmakedepends="extra-cmake-modules gettext qt6-tools qt6-base
+ qt6-wayland-tools wayland-devel kf6-kdoctools-devel kf6-kconfig-devel"
+makedepends="kf6-kdeclarative-devel kf6-kxmlgui-devel
+ kf6-knewstuff-devel kwayland-devel kf6-purpose-devel
+ xcb-util-cursor-devel xcb-util-image-devel
+ layer-shell-qt-devel kpipewire-devel
+ pipewire-devel wayland-devel qt6-multimedia-devel"
+#depends="qt5-tools"
```

Comment:
qt*-tools shouldn't be needed anymore, they removed the dependency on qdbus: https://invent.kde.org/graphics/spectacle/-/commit/8d845e429f752d7feb0a6136463d1c4ca09ecb2c

Reply:
Yes